user_guide_src/source/conf.py

```diff
@@ -53,7 +53,7 @@
 # The short X.Y version.
 version = '4.0-dev'
 # The full version, including alpha/beta/rc tags.
-release = '4.0.0-rc.4'
+release = '4.0.2'
```

Comment:
4.0.3 for next release

Reply:
Hi, I based it upon the admin/version release script. For some reason it didn't run (so I stayed on the current version), but should be run. So 4.0.3. will be added automatically. Or whatever version we bump it up too.
https://github.com/codeigniter4/CodeIgniter4/blob/develop/admin/release#L50